python-urllib3.spec

```diff
@@ -216,6 +216,9 @@ rm -rf %{buildroot}/%{python3_sitelib}/__pycache__*
 %endif # with_python3
 
 %changelog
+* Wed Sep 13 2017 Jon Schlueter <jschluet@redhat.com> 1.16-2
+- fix bad if guard
```

Comment:
This is not worth changelog entry, these are end-user documentation and this is internal build issue.
BTW I wonder how CBS build worked?